### ISSUE TITLE
[Request] Improvement to import tiddler

### ISSUE THREAD

**Issue by giffmex** [Jermolene/TiddlyWiki5#4695]
Pasted from my comment at the Google Group forum:

I think the import tiddler should have two more checkboxes: Check/uncheck system tiddlers, and check/uncheck regular tiddlers.

I say this because I suspect most users, when importing, are in one of two scenarios:

1. They want to move their notes (regular tiddlers) to a new TiddlyWiki setup, say, from Stroll to Drift, or vice versa, or from their old TiddlyWiki to TiddlyRoam, or whatever.
2. They have perfected their setup ($:/ prefixed tiddlers) in a TiddlyWiki file they started that has lots of notes, and they want to create a new template from the shadow tiddlers without importing their notes.

Either way, the simplest procedure SHOULD be,

a) download an empty.html from tiddlywiki.com or Stroll, etc
b) Import tiddlers from old file, with the ability to choose quickly to import only regular tiddlers or only $:/ prefixed tiddlers.

But to do b, one must presently check or uncheck what could be dozens or even hundreds of tiddlers.

I know there is another way to do this, exporting tiddlers into json from one file and importing into another. But this is a multistep process not easily discerned by new users, and even for experienced users, it seems like this ought to be available at the import tiddler, to save people time. And it doesn't seem like it would be that hard to use checkboxes to distinguish and filter out tiddlers that start or don't start with $:/

I think this should be core, but as the title states, I'd be willing to settle for a plugin.

**Comment by pmario** [comment]
probably related to #4263 and may be others.

**Comment by pmario** [comment]
>Github issues, where ideas go to die...

No, where ideas have to be reviewed from time to time and need to get "bumped" by users with new comments, so they won't be forgotten. 

-------

I did change the import behaviour for the bundler-plugin. As seen in the[bundler video](https://youtu.be/No3cOcRlLpo?t=492) there is a search filter, which **should** allow us to filter the list. ... BUT it seems to be broken at the moment. At least it seems so as I did test it 2 minutes ago :/
